public/pageloader.js

```diff
@@ -0,0 +1,6 @@
+chrome.browserAction.onClicked.addListener(function () {
```

Comment:
I can't help you much, because I don't know Vue, so I can't see what problems you may have.

Here I'd tell you to use the Arrow Functions: https://www.freecodecamp.org/news/when-and-why-you-should-use-es6-arrow-functions-and-when-you-shouldnt-3d851d7f0b26/

Reply:
ya it was copy pasta from developer.chrome.com, its the way they did it.

I was working on getting something working before going back for clean up.
Hate rushing right at the end just to commit broken/bad code lol

Reply:
also there really isn't any code in the vue framework yet that does anything.

Reply:
It is logical, we have all been fair with time this time. :)